readme.md

```diff
@@ -44,6 +44,11 @@ Renovate is widely used in the developer community:
 
 ![Renovate Matrix](https://app.renovatebot.com/images/matrix.png)
 
+## Who contributes to Renovate?
```

Comment:
We already have a section on contributing. I think we should move this part there. 😉 

> ## Contributing
>
> If you want to contribute to Renovate or get a local copy running, please read the instructions in [.github/contributing.md](.github/contributing.md).
> To get started look at the [list of good first issues](https://github.com/renovatebot/renovate/contribute).

Reply:
The OSS Insight page isn't purely about contributions though

Reply:
So maybe this is broad enough?  🤔 
```suggestion
## Who is the Renovate community? 
```

Reply:
How about "Renovate OSS Insights"?

Reply:
Then it is decided ^^